packages/cardano-services/test/Program/services/ogmios.test.ts

```diff
@@ -282,7 +282,7 @@ describe('Service dependency abstractions', () => {
             headers: { 'Content-Type': APPLICATION_JSON }
           });
           expect(res.status).toBe(200);
-          expect(res.data).toEqual(healthCheckResponseMock());
+          expect(res.data).toEqual(healthCheckResponseMock({ withTip: true }));
```

Comment:
I think `withTip: true` could be a default behavior of `healthCheckResponseMock()` rather than this repetition, since now `DbSyncProvider's` [healthCheck](https://github.com/input-output-hk/cardano-js-sdk/pull/553/files#diff-4737374e2ac27a76e30bc8c8858b7890aee4d442cffc3fe03ce9e9ebe77b0247R25-R42) contains the `projectedTip` and it's reused across all providers (they extend it) @iccicci

Reply:
ea59ed748 and 0576735d8 should solve this